### PR TITLE
Finish sliders and cards

### PR DESCRIPTION
**Pull request by hanqidS2023**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@ant-design/icons": "^5.2.6",
+    "@babel/plugin-proposal-private-property-in-object": "^1.0.0",
```
Comment (congruiyS2023) [inline package.json]: What's this? I got an error for this when `npm install`
``` bash
❯ npm install
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @babel/plugin-proposal-private-property-in-object@^1.0.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

File: src/components/RecycleOrderHistoryCard.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { ShoppingCartOutlined, RightOutlined } from "@ant-design/icons";
+import { Card } from "antd";
+
+const RecycleOrderHistoryCard = ({ clickRecycleHistoryCard, ...props }) => (
+  <Card
+    title={
+      <div>
+        <ShoppingCartOutlined className="text-3xl text-primary" />
+        <span className="text-primary ml-2">Recycle Order 1</span>
```
Comment (congruiyS2023) [inline src/components/RecycleOrderHistoryCard.jsx]: Heading font

File: src/components/RedistributeCard.jsx
Comment (congruiyS2023) [inline src/components/RedistributeCard.jsx:1]: - Please use font specified in the style guide. See Figma for font specification.
- Could you please make all text left align as in the Figma style guide?

**Review by congruiyS2023 — CHANGES_REQUESTED** [review]

File: src/components/RecycleOrderHistoryCard.jsx (hunk 2 above)
Comment (hanqidS2023) [reply src/components/RecycleOrderHistoryCard.jsx]: done

File: package.json (hunk 1 above)
Comment (hanqidS2023) [reply package.json]: removed

File: src/components/RedistributeCard.jsx
Comment (hanqidS2023) [reply src/components/RedistributeCard.jsx:1]: done

**Comment by hanqidS2023** [comment]
Screenshot 2023-12-01 at 12 11 43 AM

**Review by congruiyS2023 — APPROVED** [review]
Great, could have more unified font size on card title but I guess we can live with that.

**Pull request merged**